scapy/layers/ipsec.py

```diff
@@ -111,7 +113,7 @@ class ESP(Packet):
     name = 'ESP'
 
     fields_desc = [
-        XIntField('spi', 0x0),
+        XIntField('spi', 0xdeadbeef),
```

Comment:
How about 1?

Reply:
I changed it to ` 0x00000001` , see 09d0590f48a52f1f950f4239c0caee61beba1f27.